src/test/java/org/primeframework/mvc/security/cookiesession/MockUser.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc.security.cookiesession;
+
+import java.util.UUID;
+
+public class MockUser implements IdentifiableUser {
+  public UUID id;
+  public String email;
```

Comment:
Let's format all of these files to match the IJ project style.

Reply:
Done